isis/CMakeLists.txt

```diff
@@ -150,6 +150,10 @@ if(JP2KFLAG)
  set(JP2KFLAG 1)
 endif()
 
+# Set up the ctest tool which is used to run all of the tests.
```

Comment:
These need to be in an `if(buildTests)` block like the very end of this file. Also, remove them from the very end of the file, so all that block does is add the `tests` subdirectory.